src/backend/opengl/CommandBufferGL.cpp

```diff
@@ -106,15 +106,21 @@ namespace opengl {
 
                         const auto& info = currentRenderPass->GetSubpassInfo(currentSubpass);
 
-                        for (uint32_t index = 0; index < info.colorAttachments.size(); ++index) {
-                            uint32_t attachment = info.colorAttachments[index];
+                        std::array<GLenum, kMaxColorAttachments> drawBuffers = {};
```

Comment:
Can we explicitly fill this with GL_NONE even if it is 0. Or add a comment explaining why the zero init does what we want?